src/styles/card.css

```diff
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
```

Comment:
not sure if it's necessary, but you could assign colours to a variables and using them that way. but again, there's no issue, just a suggestion